### DISCUSSION TITLE
College Professor Rejected from GitHub Education

### DISCUSSION THREAD

**Discussion by alexwgold** [community/community#148219] [GitHub Education]
### Hi everyone,

I'm a college professor and attempting to register for Github Education.

I got my college email set up, which should be proof enough.  But my college doesn't do ID cards, so I sent in a picture of my contract for this coming semester as proof of employment there.

It was automatically rejected, though that should be sufficient proof. Is there really no way to contact Github and ask for a manual review?

**Comment by AuroraEchos** [comment]
Hey brother, I understand you. Although you have registered using your school email and submitted this semester's work contract as proof of employment, your application was automatically rejected. Perhaps GitHub works this way, we can only follow their requirements on the application website, and the work contract is not recognized. On the support page, you may find a form to submit questions or requests, or a dedicated support email address. I suggest you use these channels again to submit your question and attach your school email address, a photo of your work contract, and any other relevant supporting documents.

**Reply by samus-aran** [reply]
Work contractors are accepted 👍 

All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

**Comment by samus-aran** [comment]
Hi @alexwgold 👋 

What were your rejection reasons? They provide what you need to consider when re-applying. I suggest you enable 2FA first and try again.
